bin/lib/vets-api/setups/docker.rb

```diff
@@ -19,18 +19,23 @@ def run
       private
 
       def configuring_clamav_antivirus
-        print 'Configuring ClamAV in local settings...'
-        file_path = 'config/settings.local.yml'
-        data = YAML.load_file(file_path)
+        print 'Updating settings.local.yml...'
+        settings_path = 'config/settings.local.yml'
+        settings_file = File.read(settings_path)
+        settings = YAML.safe_load(settings_file, permitted_classes: [Symbol])
+        settings.delete('clamav')
 
-        data['clamav'] = {
+        settings['clamav'] = {
```

Comment:
Do you want to add "print 'Configuring ClamAV in local settings...'" back right here?